mandatory/4-tax.js

```diff
@@ -1,11 +1,18 @@
-/*
-  SALES TAX
-  =========
-  A business requires a program that calculates how much the price of a product is including sales tax
-  Sales tax is 20% of the price of the product.
-*/
+/**
+ * /*
+ *   SALES TAX
+ *   =========
+ *   A business requires a program that calculates how much the price of a product is including sales tax
+ *   Sales tax is 20% of the price of the product.
+ *
+ * @format
+ */
 
-function calculateSalesTax() {}
+//Sales Tax function adds 20% of the price
+function calculateSalesTax(price) {
+	var newPrice = price + (20 / 100) * price;
+	return newPrice;
+}
```

Comment:
This works fine. One change you could make is to declare 20, or 20/100 as a variable called taxRate or something similar, which would make it easier to understand what the code is doing. However, your method has the advantage of being more concise, so it's a matter of taste really, both are correct.